src/max-width.js

```diff
@@ -0,0 +1,27 @@
+const { get, is, px, num, breaks, dec, media, merge } = require('./util')
+const { maxWidths } = require('./constants')
```

Comment:
I've been leaning towards fewer (potentially none) default values in styled-system – thinking it's better to work off of raw values, then let users set their own values in their theme – and I think max-widths are one of those things that probably vary widely from project to project